src/pages/DashBoard/DashBoard.js

```diff
@@ -1,8 +1,116 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import ItemCard from '../../components/ItemCard/ItemCard';
+import AdCard from './AdCard/AdCard';
 import './DashBoard.scss';
+import UseALink from './UseALink/UseALink';
 
 const DashBoard = props => {
-  return <>DashBoard</>;
+  const [items, setItems] = useState([]);
+  const [recommendItems, setRecommendItems] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/perfumes.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        setItems(data);
+      });
+
+    fetch('/data/recommends.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        setRecommendItems(data);
+      });
+  }, []);
+
+  return (
+    <div className="dashBoard">
+      <div className="dashBoardWrapper">
+        <div className="dashBoardImg">
+          <div className="dashBoardInfo">
+            <header>
+              <h2>호기심으로 물든 일곱 가지 아로마</h2>
+              <h1>몸과 마음에 머무는 아로마</h1>
+            </header>
+            <p>
+              자아와 감각을 깨우는 아로마의 이솝 오 드 퍼퓸은 조향에 대한
+              일반적인 관행을 따르지 않는 미묘한 차이의 접근법으로 만들어집니다
+            </p>
+            <div className="goToPerfume">
+              <UseALink theme="white" text="향수보기" />
+            </div>
+          </div>
+        </div>
+      </div>
+      <section>
+        <div className="fragranceWrapper">
+          {items.map(list => {
+            return (
+              <ItemCard
+                key={list.id}
+                itemPhoto={list.img}
+                itemName={list.name}
+                itemDescription={list.feature_name}
+              />
```

Comment:
```suggestion
              <ItemCard
                key={list.id}
                list={list}
              />
```
이렇게 list 전체를 넘겨주고, ItemCard 컴포넌트에서 받아서 사용할 때 구조분해할당한 후 사용해주세요!

Reply:
이 부분이 이해가 안돼서 일단은 다른구조분해할당을 적용했습니다. 만나서 질문하고싶습니다 ☺️